core/src/services/webdav/fixtures/docker-compose-webdav-with-auth.yml

```diff
@@ -23,7 +23,7 @@ services:
     ports:
       - "8080:80"
     volumes:
-      - ./data:/data
+      - ./webdav-data:/data
```

Comment:
Hi, here is some misunderstanding.

My previous comment is `webdav-data:/data` instead of `./webdav-data:/data`. They are different behavior at docker-compose side and don't need to create a webdav-data folder by hand.